packages/alfa-css/src/value/shadow.ts

```diff
@@ -99,7 +99,7 @@ export class Shadow<
     this._blur.hash(hash);
     this._spread.hash(hash);
     this._color.hash(hash);
-    Hash.writeBoolean(hash, this._isInset);
+    hash.writeBoolean(this._isInset);
```

Comment:
Would it makes sense to have that also chainable, likely with a
```javascript
writeHashable(foo: Hashable) {
  foo.hash(this);
  return this;
}
```
and then we can turn this (and similar) to
```javascript
hash.writeHashable(this._blur).writheHashable(this._spread).writeHashable(this._color).writeBoolean(this.isInset)
```

Reply:
Oh, I actually already added that but forgot to use it:

https://github.com/Siteimprove/alfa/blob/4d3955dfa86b632f7153998b0b86005ad949e090/packages/alfa-hash/src/hash.ts#L104-L107